app/assets/stylesheets/snowcrash/modules/notifications.scss

```diff
@@ -0,0 +1,78 @@
+.notifications {
+  .notifications-dot {
+    display: none;
```

Comment:
I liked this dot.
Can't we make it work?

Reply:
I just added this dot to show how it looks on the navbar. The functionality should be added with the ActionCable feature.